cli/src/commands/git/init.rs

```diff
@@ -161,8 +161,7 @@ fn do_init(
         if colocated_git_repo_path.exists() {
             return Err(user_error_with_hint(
                 "Did not create a jj repo because there is an existing Git repo in this directory.",
-                "To create a repo backed by the existing Git repo, run `jj git init --colocate` \
-                 instead.",
+                "To create a repo backed by the existing Git repo, run `jj git init` instead.",
```

Comment:
I think we should suggest `--colocate` here because it's more likely that the user ran `jj git init` with `git.colocate=false`.